### PR TITLE
team.bot_settings link

### PR DESCRIPTION
**Pull request by joshblum**
cc @keybase/hotpotatosquad

### CODE REVIEW

File: src/schema3.iced [hunk 1]
```diff
@@ -167,7 +167,6 @@ exports.enc_kid = () -> new KID { encryption : true }
 exports.seqno = () -> new Seqno {}
 exports.time = () -> new Time {}
 exports.int = () -> new Int {}
-exports.uid = () -> new Binary { len : 16 }
```
Comment (joshblum) [inline src/schema3.iced]: dupe of L164

**Comment by joshblum** [comment]
cc @maxtaco can you publish when you get a chance 🙏

**Comment by maxtaco** [comment]
@joshblum I don't have this setup from home, can I do it tomorrow?

**Comment by joshblum** [comment]
no rush, thanks!

**Pull request merged**